bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/README.md

```diff
@@ -0,0 +1,87 @@
+#HBase Tools
+
+This folder contains tools to help HBase users with migrations to Cloud Bigtable.
+Pipelines to import and export data are under [bigtable-beam-import](bigtable-dataflow-parent/bigtable-beam-import/README.md).
+
+## Setup 
+
+To use the tools in this folder, you can download them from the maven repository, or
+you can build them using Maven.
+
+[//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
+### Download the jars
+
+Download [the Bigtable tools jars](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-hbase-1.x-tools/1.14.1/bigtable-hbase-1.x-tools-1.14.1-shaded.jar), which is an aggregation of all required jars.
+
+### Build the jars
+Go to the top level directory and build the repo then return to this sub directory.
+
+```
+cd ../../
+mvn clean install -DskipTests=true
+cd bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools
+```
+
+## Schema Translation tool 
+This tool will create tables in Cloud Bigtable based on the tables in an HBase cluster.
+You specifiy a [name regex](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html?is-external=true)
+and it will copy column families, garbage collection rules,
+and table splits.
+
+1. Define the environment variables to easily run the command.
+    ```
+    PROJECT_ID=your-project-id
+    INSTANCE_ID=your-instance-id
+    TABLE_NAME_REGEX=your-table-name
+    
+    ZOOKEEPER_QUORUM=localhost
+    ZOOKEEPER_PORT=2181
+    ```
+1. Execute the following command to copy the schema from HBase to Cloud Bigtable.
+    ```
+    java \
+     -Dgoogle.bigtable.project.id=$PROJECT_ID \
+     -Dgoogle.bigtable.instance.id=$INSTANCE_ID \
+     -Dgoogle.bigtable.table.filter=$TABLE_NAME_REGEX \
+     -Dhbase.zookeeper.quorum=$ZOOKEEPER_QUORUM \
+     -Dhbase.zookeeper.property.clientPort=$ZOOKEEPER_PORT \
+     -jar bigtable-hbase-1.x-tools-1.14.1-SNAPSHOT-jar-with-dependencies.jar
+    ```
+
+### Alternative: Exporting Schema
+
+If your HBase master is in a private VPC or can't connect to internet, you can
+export the HBase schema to a file and use that to create tables in Cloud Bigtable.
+
+
+#### Export schema
+
+1. On a host that can connect to HBase, define the export location for your schema file.
```

Comment:
Should we mention that schema file is a JSON file? and a concrete example could be `/tmp/schema.json`